app/services/api.js

```diff
@@ -0,0 +1,6 @@
+import axios from 'axios';
```

Comment:
I'm not sure that creating new `services` only for this file is worth it. Can we move this file either to `config` or `utils` directories for the sake of simplicity?

Reply:
To be honest it's not only for this file. There's a real need for this directory in my opinion. Often there're other services, like `contentful` or `firebase` and they are neither a util nor config. I always struggled with a decision on where to put them.
The only argument would be because we don't want another directory, but that doesn't seem convincing.

I could add it to README to specify what exactly is a role of this directory.

Reply:
ok